static.md

```diff
@@ -50,38 +50,34 @@ flag is provided. If you want to compact files, provide the `compact` option in
 your configuration. It is an object that maps file extensions to functions. The
 functions are expected to receive a buffer as their only argument, returning
 either a buffer or a promise wrapped around a buffer. The resulting buffer is
-the compacted version of the provided buffer. Example:
+the compacted version of the provided buffer.
+
+A common use case is the compaction of images. The following configuration is
+a good starting point (previously available via the `compact: "images"`
```

Comment:
"previously" meaning "before v1.2.0"? (v2.0.0?)

Reply:
I don't know yet, but this will be a breaking change, so probably 2.0.0? 😄 It is deprecated, so people should stop using it. How would you describe that?

Reply:
> (the equivalent `compact: "images"` shortcut is deprecated, and will be removed in a future version)
?

Reply:
Yeah, that sounds good - perhaps like this:

````
A common use case is compressing images. The following configuration is a good
starting point:

'''js
…
'''

(the equivalent `compact: "images"` shortcut is deprecated and will be removed in a future version)
```

(no comma 😛 )

Reply:
Adjusted 👍